packages/eslint-config/index.js

```diff
@@ -3,11 +3,15 @@
 // Framework main config is in monorepo root ./.eslintrc.js
 
 const {
-  getConfig,
+  getApiSideDefaultBabelConfig,
```

Comment:
It might be a good idea to add a lint rule to prevent `import * from XXX` though I know those types of imports are shotgunned through the app -- that way you at least warn and force an override when folks want to do that

Reply:
The PR already includes the lint rules!